src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: ($day-size + $gap-columns) * $columns;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), arithmetic operations should be enclosed in parentheses to improve readability. Consider wrapping the entire arithmetic operation for 'max-width' in parentheses.